<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>256</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="randomButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Случайные значения</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeSelectBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>пузырьком</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>быстрая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>расческа</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>гномья</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>обезьянья</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="reorderButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Навести порядок</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>193</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="rowCount_Label">
      <property name="text">
       <string>Кол-во строк</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="rowCount_LineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="search_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_LineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="vertexLabels">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="maxButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>MAX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="avgButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>AVG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>MIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_Label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deliteButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить дубликаты</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>261</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Найдено элементов:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_seeked">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
